docs/swagger.yaml

```diff
@@ -598,9 +577,8 @@ components:
           $ref: '#/components/schemas/ShippingInfo'
         paymentSources:
           $ref: '#/components/schemas/PaymentSources'
-        receiptNumber:
-          type: string
-          nullable: true
+        barcode:
+          $ref: '#/components/schemas/MobileBarcode'
```

Comment:
Rename to just "barcode" here. 
The components are prefixed with mobile in the mobile api so we can seperate the autogenerated classes in the code :)